packages/terra-tag/src/_RollupTag.jsx

```diff
@@ -74,7 +74,7 @@ const RollUpTag = (props) => {
       onBlur={handleOnBlur}
       refCallback={(ref) => { rollUpTagRef.current = ref; }}
       isCompact
-      aria-live={(isCollapsed) ? 'polite' : 'off'}
+      aria-live={(navigator.userAgent.indexOf('Edg') === -1 && isCollapsed && !/^(?=.*\bSafari\b)(?!.*\bChrome\b).*/i.test(navigator.userAgent)) ? 'polite' : 'off'}
```

Comment:
aria-live is now used only when its either safari or chrome ? Can the expression be simplified ?

Reply:
@gt106551  What does this expression validate to ..??

Reply:
I have tried to simplify in different ways but getting issues in safari browser

Reply:
> @gt106551 What does this expression validate to ..??

it checks if the user agent string contains 'Safari' but does not contain 'Chrome'.

Reply:
If you need only safari, refer : https://github.com/cerner/terra-core/blob/8220010acd80830e9bd7368ebbf7bc694d731e8a/packages/terra-form-select/src/shared/_SharedUtil.js#L7
Include edge as well in this

Reply:
I think you're trying to set aria-live to `polite` for edge and JAWS. In this case you can refer to some of the existing regEx for edge browser instead of adding new expression from web.

Reply:
I Have removed  aria-live attribute and tested in all browsers it is working as expected.